scannerpython.properties

```diff
@@ -0,0 +1,40 @@
+category=Scanners
+name=SonarScanner for Python
+license=GNU LGPL 3
+organization=SonarSource
+organizationUrl=https://www.sonarsource.com
+homepageUrl=https://docs.sonarsource.com/sonarqube-server/latest/analyzing-source-code/scanners/sonarscanner-for-python/
+issueTrackerUrl=https://sonarsource.atlassian.net/jira/software/c/projects/SCANPY/issues
+sourcesUrl=https://github.com/SonarSource/sonar-scanner-python
+archivedVersions=0.1.0.285,0.1.0.340,0.2.0.520,1.0.0.1453,1.0.1.1548
```

Comment:
Isn't version `0.2` still valid for SQ instances that do not support bootstrapping?

Reply:
yep, confirmed with Ann, no need have two versions.